agent/functional_tests/tests/functionaltests_test.go

```diff
@@ -422,6 +423,39 @@ func TestCustomAttributesWithMaxOptions(t *testing.T) {
 	assert.True(t, ok, "OS attribute not found")
 }
 
+// containerHealthMetricsTest tests the container health metrics based on the task definition
+func containerHealthMetricsTest(t *testing.T, taskDefinition string) {
+	agent := RunAgent(t, nil)
+	defer agent.Cleanup()
+
```

Comment:
You should probably add `agent.RequireVersion(>=1.17.0)`.

Reply:
Are you planning to address this in this PR?

Reply:
I will add this along with the version change in dev branch when it was merged back to dev.